### DISCUSSION TITLE
Welcome to ZenTimings Discussions!

### DISCUSSION THREAD

**Discussion by irusanov** [irusanov/ZenTimings#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BiuMinato** [comment]
<img width="1080" height="1920" alt="zentimingbaocuo" src="https://github.com/user-attachments/assets/e2256d18-36ce-42e2-8d2a-e40bb23f7e4a" />
Every time the pawnio driver is restarted, it will go offline, including smudebugtool will also report an error and cannot be used. Only after uninstalling and reinstalling can it return to normal, but after several restarts, it can't be used and needs to be reinstalled.
